projects/packages/forms/src/dashboard/state/actions.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Internal dependencies
+ */
+import {
+	JETPACK_FORMS_RESPONSES_FETCH,
+	JETPACK_FORMS_RESPONSES_FETCH_RECEIVE,
+	JETPACK_FORMS_RESPONSES_FETCH_FAIL,
+} from './action-types';
+import responses from './fixtures/responses';
+
+export const fetchResponses = ( query, limit = 20, offset = 0 ) => {
+	return async dispatch => {
+		dispatch( {
+			type: JETPACK_FORMS_RESPONSES_FETCH,
+			query,
+			limit,
+			offset,
+		} );
+
+		const queryString = new URLSearchParams( {
+			...query,
+			limit,
+			offset,
+		} ).toString();
+
+		try {
```

Comment:
Now that I think of it, a simpler way of dealing with this would be to put the request logic inside a hook, say `useResponseQuery` or something, and just have that call the necessary actions instead of putting the fetch directly inside the state layer.

But we need to revisit this anyway to connect to the real api endpoint so I'm fine if we do it in a later change.